app/views/users/index.html.erb

```diff
@@ -1,3 +1,18 @@
-<div>
-     <h1> Here is a list of all users <h1>
-<div />
\ No newline at end of file
+<section class="show-user">
+   <div class="card" style="width: 18rem;">
+      <ul class="users_list">
+          <% @users.each do |user| %>
+               <li class = 'user' >
+               <% if user.photo %>
+                    <img class="card-img-top" src='<%= user.photo %>' alt='<%= user.name %> thumbnail' />
+               <% end %>
+                    <div class='card-body'>
+                         <h5> Username: <%= user.name%></h5>
+                         <p class="card-text">Number of Posts: <%= user.posts_counter %></p>
+                          <%= link_to 'View user posts', user_path(user), class: "btn btn-primary text-center"  %>
```

Comment:
- [ ] Again as requested by the prvious reviewer, Kindly make sure the app/views/users/show.html.erb view page match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) as required for the project

Your design
![image](https://user-images.githubusercontent.com/81932478/214284261-d64579cd-039d-4703-8e26-24e4cd0e24fa.png)
![image](https://user-images.githubusercontent.com/81932478/214284260-f537bd50-f9c8-45fa-8487-09fb945c8e24.png)

Given frame
![image](https://user-images.githubusercontent.com/81932478/214284530-01c3e5a5-fe1c-4eaa-a409-656faad8b006.png)